crates/pipeline_manager/src/db/mod.rs

```diff
@@ -329,6 +338,12 @@ pub(crate) struct ConnectorDescr {
     pub direction: Direction,
 }
 
+#[derive(Serialize, ToSchema, Debug, Clone, Eq, PartialEq)]
```

Comment:
add docs?

Reply:
Added docs and also changed the name to something clearer: ApiPermission